Week2/homework/2theevenoddreporter.js

```diff
@@ -0,0 +1,9 @@
+// Report whether or not a number is odd/even!
+//
+// Create a for loop, that iterates from 0 to 20.
+// Create a conditional statement that checks if the value of the counter variable is odd or even.
+// If it's odd, log to the console The number [PUT_NUMBER_HERE] is odd!.
+// If it's even, log to the console The number [PUT_NUMBER_HERE] is even!.
+for(let i = 0; i < 20; i++){
+  console.log(i%2===1 ? `${i} is odd!` : `${i} is even!`);
+}
```

Comment:
Ah. I've two things to point here:
1. You were required to run the loop from 0 till 20 but your loop would run only until 19. Can you check why?
2. Ideally, we should use conditional operators i.e `? :` when we want to do an assignment operation with the values from conditional expression. For example, `const isItCold = weather < 20 ? 'Yes' : 'No'`. The usage of conditional for your case is just an expression i.e printing a statement. I would suggest you use the usual `if-else` block here for such operations. 🙂

Reply:
1. I did not use the <= operator. 
2. I could have done that, this seemed shorter to write.
Would it be okay if i do it hits way then?
let stts = i%2===1 ? `${i} is odd!` : `${i} is even!`;
console.log(stts);